pymc3/distributions/mixture.py

```diff
@@ -36,8 +36,32 @@ class Mixture(Distribution):
     w : array of floats
         w >= 0 and w <= 1
         the mixture weights
-    comp_dists : multidimensional PyMC3 distribution or iterable of one-dimensional PyMC3 distributions
-        the component distributions :math:`f_1, \ldots, f_n`
+    comp_dists : multidimensional PyMC3 distribution (e.g. `pm.Poisson.dist(...)`)
+        or iterable of one-dimensional PyMC3 distributions the
+        component distributions :math:`f_1, \ldots, f_n`
+
+    Example
+    -------
+    # 2-Mixture Poisson distribution
+    with pm.Model() as model:
+        lam = pm.Exponential('lam', lam=1, shape=(2,))  # `shape=(2,)` indicates two mixtures.
+        components = pm.Poisson.dist(mu=lam, shape=(2,))  # must use dist, not plain Poisson object!
```

Comment:
I would adjust the comment a bit, maybe: `# as we just need the logp, rather than add a RV to the model, we need to call  `dist`

Reply:
Done!